src/generated/resources/assets/advancedperipherals/lang/en_us.json

```diff
@@ -59,6 +60,7 @@
   "item.advancedperipherals.tooltip.peripheral_casing": "&7An empty hull without the love it deserves. Used as a crafting ingredient",
   "item.advancedperipherals.tooltip.player_detector": "&7This peripheral can be used to interact with players, but don't be a stalker.",
   "item.advancedperipherals.tooltip.redstone_integrator": "&7This block is able to interact with redstone. Works exactly like the redstone api of an computer.",
+  "item.advancedperipherals.tooltip.rs_bridge": "&7The RS Bridge interacts with Refined Storage 2 to manage your items.",
```

Comment:
idk if we need specifically mention `Refined Storage *2*` here, since below the id is `refinedstorage` not `refinedstorage2`

Reply:
Mhh yeah, you are prob right there